examples/showcase/controller-specific-anchors/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
This is a bit too specific to be part of the showcase that should be more like self-contained mini-apps

Reply:
We have a watch demo. What's missing there? https://github.com/aframevr/aframe/tree/master/examples/mixed-reality/watch